tests/restapi/test_restapi.py

```diff
@@ -0,0 +1,192 @@
+import pytest
+import time
+import logging
+import requests
+import json
+
+from tests.common.helpers.assertions import pytest_assert
+from restapi_operations import Restapi
+
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.topology('t0'),
+    pytest.mark.disable_loganalyzer
+]
+
+CLIENT_CERT = 'restapiclient.crt'
+CLIENT_KEY = 'restapiclient.key'
+
+restapi = Restapi(CLIENT_CERT, CLIENT_KEY)
+
+'''
+This test creates a default VxLAN Tunnel and two VNETs. It adds VLAN, VLAN member, VLAN neighbor and routes to each VNET
+'''
+def test_data_path(construct_url):
+    # Create Default VxLan Tunnel
+    params = '{"ip_addr": "10.3.152.32"}'
+    logger.info("Creating Default VxLan Tunnel with ip_addr: 10.3.152.32")
+    r = restapi.post_config_tunnel_decap_tunnel_type(construct_url, 'vxlan', params)
+    pytest_assert(r.status_code == 204)
+
+    # Check RESTAPI server heartbeat
+    logger.info("Checking for RESTAPI server heartbeat")
+    restapi.heartbeat(construct_url)
+
+    #
+    # Create first VNET and add VLAN, VLAN member, VLAN neighbor and routes to it
+    #
+
+    # Create VNET
+    params = '{"vnid": 2000}'
+    logger.info("Creating VNET vnet-guid-2 with vnid: 2000")
+    r = restapi.post_config_vrouter_vrf_id(construct_url, 'vnet-guid-2', params)
+    pytest_assert(r.status_code == 204)
+
+    # Verify VNET has been created
+    r = restapi.get_config_vrouter_vrf_id(construct_url, 'vnet-guid-2')
+    pytest_assert(r.status_code == 200)
+    logger.info(r.json())
+    expected = '{"attr": {"vnid": 2000}, "vnet_id": "vnet-guid-2"}'
+    pytest_assert(r.json() == json.loads(expected))
+    logger.info("VNET with vnet_id: vnet-guid-2 has been successfully created with vnid: 2000")
+
+    # Create VLAN
+    params = '{"vnet_id": "vnet-guid-2", "ip_prefix": "100.0.10.1/24"}'
+    logger.info("Creating VLAN 2000 with ip_prefix: 100.0.10.1/24 under vnet_id: vnet-guid-2")
+    r = restapi.post_config_vlan(construct_url, '2000', params)
+    pytest_assert(r.status_code == 204)
+
+    # Verify VLAN has been created
+    r = restapi.get_config_vlan(construct_url, '2000')
+    pytest_assert(r.status_code == 200)
+    logger.info(r.json())
+    expected = '{"attr": {"ip_prefix": "100.0.10.1/24", "vnet_id": "vnet-guid-2"}, "vlan_id": 2000}'
+    pytest_assert(r.json() == json.loads(expected))
+    logger.info("VLAN 2000 with ip_prefix: 100.0.10.1/24 under vnet_id: vnet-guid-2 has been successfully created")
+
+    # Add and configure VLAN member
+    params = '{"tagging_mode": "tagged"}'
+    logger.info("Adding Ethernet216 with tagging_mode: tagged to VLAN 2000")
```

Comment:
Ethernet216 interface may not be available on all SKUs. So when we run this on 2700 T0s, it will fail. My suggestion is to get the Vlan member data and use the first member port. Similar to what is done for vnet tests.